test/PushNotificationTests/BaseTestSuite.cpp

```diff
@@ -113,6 +115,27 @@ void BaseTestSuite::ChannelRequestUsingRemoteId()
     }
 }
 
+void BaseTestSuite::ChannelRequestCheckExpirationTime()
+{
+    if (PushNotificationManager::Default().IsSupported())
+    {
+        auto channelOperation{ PushNotificationManager::Default().CreateChannelAsync(c_azureRemoteId) };
+        VERIFY_SUCCEEDED(ChannelRequestHelper(channelOperation));
+
+        auto channel{ channelOperation.GetResults().Channel() };
+        auto expirationTime{ channel.ExpirationTime() };
+        auto expiryBound{ winrt::clock::now() + (hours(24) * 30) + minutes(1) };
+
```

Comment:
You checked the upper bound but forgot to check the lower bound

Reply:
This means that if Expiry time returns 1800/01/01, the test would pass incorrectly

Reply:
The lower bound would be the current time.